pkg/kubelet/kubelet_test.go

```diff
@@ -461,6 +461,62 @@ func TestHandlePortConflicts(t *testing.T) {
 	require.Equal(t, v1.PodPending, status.Phase)
 }
 
+// Tests that we sort pods based on criticality.
+func TestCriticalPrioritySorting(t *testing.T) {
```

Comment:
If it turns out that we *do* want a criticality sort that is stable wrt creation time, we should check that in this test.